docs/walkthrough/integrate-with-jina.md

```diff
@@ -0,0 +1,72 @@
+(integrate-with-jina)=
+# Integrate with Jina
+
+Once fine-tuning is finished, it's time to actually use the model. 
+Finetuner, being part of the Jina ecosystem, provides a convenient way to use tuned models via [Jina Executors](https://docs.jina.ai/fundamentals/executor/).
+
+We've created the [`FinetunerExecutor`](https://hub.jina.ai/executor/13dzxycc) which can be added in a [Jina Flow](https://docs.jina.ai/fundamentals/flow/) and load any tuned model. 
+
+````{tab} via Docker image (recommended)
+```python
+from jina import Flow
+	
+f = Flow().add(uses='jinahub+docker://FinetunerExecutor', uses_with=...)
+```
+````
+````{tab} via source code
+```python
+from jina import Flow
+	
+f = Flow().add(uses='jinahub://FinetunerExecutor', uses_with=...)
+```
+````
+
+## Using [`FinetunerExecutor`](https://hub.jina.ai/executor/13dzxycc)
+
+Loading a tuned model is simple! You just need to provide a few parameters under the `uses_with` argument when adding the `FinetunerExecutor` to the [Flow]((https://docs.jina.ai/fundamentals/flow/)).
+
+Here's the full argument list of the `FinetunerExecutor`:
+
+```{admonition} FinetunerExecutor parameters
+:class: tip
+The only required argument is `artifact`. We provide default values for others.
+```
+
+* `artifact`: Specify a finetuner run or model artifact. Can be a path to a
+        local directory, a path to a local zip file or a Hubble artifact ID. (**required**)
```

Comment:
i think we need to clarify this part a bit. finetuner produce a zipped file. the local path could be this file, if user unzip it, then use this unzipped folder. Or user can get the `artifact_id` from the logs, just copy-paste the id as the `artifact` parameter